src/dawn/Unittest/PrintAllExpressionTypes.cpp

```diff
@@ -1,112 +0,0 @@
-//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
why remove this?

Reply:
because the idea is to always call this json dumper in debug mode from gtclang. So that we always call the path of code related to that. Then we dont need these tests anymore. They are not even running (to my understanding), just compiling the visitor...

Reply:
Upon more detailed looks at it, this piece seems to be completely unused. Do you agree? If so I'm fine removing it